itkwidgets/viewer.py

```diff
@@ -241,6 +242,13 @@ def set_background_color(self, bgColor: List[float]):
     def get_background_color(self):
         return self.viewer_rpc.itk_viewer.getBackgroundColor()
 
+    @fetch_value
+    def set_cropping_planes(self, croppingPlanes):
```

Comment:
nit: for the Python argument use snake case, `croppingPlanes` -> `cropping_planes`

Reply:
:100: